phpstan.neon.dist

```diff
@@ -8,3 +8,8 @@ parameters:
         - tests
     bootstrapFiles:
         - tests/bootstrap.php
+
+    ignoreErrors:
+        # Ignore errors about classes removed from doctrine/dbal 3.x
+        - '/Class Doctrine\\DBAL\\Platforms\\(MySqlPlatform|PostgreSqlPlatform) not found\./'
+        - '/Instantiated class Doctrine\\DBAL\\Driver\\PDOException not found\./'
```

Comment:
```suggestion
        - '#^Class Doctrine\\DBAL\\Platforms\\(MySqlPlatform|PostgreSqlPlatform) not found\.$#'
        - '#^Instantiated class Doctrine\\DBAL\\Driver\\PDOException not found\.$#'
```
I don't have time right now to check, but I'll try to take a look soon.
If we must keep these errors ignored, I agree they should be included in the baseline.